lib/specifier.d.ts

```diff
@@ -1,4 +1,4 @@
-import type { parsed, Range } from "./shared";
+import type { Pep440Version, Pep440Constraint } from "./shared";
 
 export const RANGE_PATTERN: string;
```

Comment:
This is also used by renovate, so we should export it from index too in a separate PR